examples/ccxt.pro/py/binance-fetch-balance-snapshot-watch-balance-updates.py

```diff
@@ -50,4 +50,5 @@ async def main():
     await exchange.close()
 
 
-run(main())
+loop = get_event_loop()
+loop.run_until_complete(main())
```

Comment:
Why was `get_event_loop` put back in for all these examples?

Reply:
https://github.com/ccxt/ccxt/pull/10573